ping.go

```diff
@@ -182,6 +182,9 @@ type Pinger struct {
 	// Source is the source IP address
 	Source string
 
+	// Iface used to send/recv ICMP messages
+	Iface string
```

Comment:
I don't like the short name
```suggestion
	// Interface used to send/recv ICMP messages
	Interface string
```

Reply:
The original intention was just to avoid confusion with the go `interface` keyword

Reply:
Maybe `InterfaceName` then if needed?